### PR TITLE
updated widget activation/deactivation hack to a more sustainable manner

### LINKED ISSUE #25: post_link script fails
```
ERROR conda.core.link:_execute(502): An error occurred while installing package 'conda-forge::nglview-1.1.9-py_0'.
LinkError: post-link script failed for package conda-forge::nglview-1.1.9-py_0
running your command again with `-v` will provide additional information
location of failed script: /srv/conda/bin/.nglview-post-link.sh
==> script messages <==
<None>

Attempting to roll back.
```

### PR DESCRIPTION
**Pull request by marscher**
fixes #25 

It is more sustainable, because it also solves the uninstall hook thing.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,12 +13,18 @@ source:
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed --install-option="--conda" .
+  script: |
+     python -m pip install --no-deps --ignore-installed --install-option="--conda" . ;
+     jupyter-nbextension install nglview --py --sys-prefix ;
+     jupyter-nbextension enable nglview --py --sys-prefix
```
Comment (hainm) [inline recipe/meta.yaml]: why does this help user installing nglview? Does this only happen during the build process, not the install process?

**Comment by marscher** [comment]
It ensures, that nglview is already activated when being installed. The activation file is just a snippet of json stored in the configuration directory. So this also solves the uninstallation issue achieved via post-uninstall

Am 25. März 2019 19:19:19 MEZ schrieb Hai Nguyen <notifications@github.com>:
>hainm commented on this pull request.
>
>
>
>> @@ -13,12 +13,18 @@ source:
> 
> build:
>   number: 0
>-  script: python -m pip install --no-deps --ignore-installed
>--install-option="--conda" .
>+  script: |
>+     python -m pip install --no-deps --ignore-installed
>--install-option="--conda" . ;
>+     jupyter-nbextension install nglview --py --sys-prefix ;
>+     jupyter-nbextension enable nglview --py --sys-prefix
>
>why does this help user installing nglview? Does this only happen
>during the build process, not the install process?

File: recipe/meta.yaml (hunk 1 above)
Comment (hainm) [reply recipe/meta.yaml]: @marscher I am still confused here. Saying you run the build process on your computer, the  `jupyter-nbextension install` and `jupyter-nbextension enable` only happen during the build process in your computer. If I install the built package to my computer, how does the `enable` happen?

Comment (marscher) [reply recipe/meta.yaml]: The enable happens via an activation file stored in the installation prefix (as well as the JS files). The file(s) is created during the build process. This way the extension is also properly uninstalled.
Does this make sense to you now?

Comment (hainm) [reply recipe/meta.yaml]: make sense to me now.

May be we already do that in setup.py:
https://github.com/arose/nglview/blob/master/setup.py#L40

Comment (hainm) [reply recipe/meta.yaml]: But it's no hurt to do it again here in this recipe to be sure.

**Pull request merged**